include/sav/Item.hpp

```diff
@@ -57,6 +60,40 @@ class Item
     virtual operator Item8(void) const;
 };
 
+class Item3 : public Item
+{
+private:
+    std::array<u8, 4> itemData;
+    u16 key;
+
+public:
+    Item3(u8* data = nullptr, u16 securityKey = 0)
+    {
+        if (data)
+        {
+            std::copy(data, data + 4, itemData.data());
+            Endian::convertFrom<u16>(itemData.data() + 2, Endian::convertTo<u16>(itemData.data() + 2) ^ securityKey);
+            key = securityKey;
+        }
+        else
+        {
+            itemData = {0, 0, 0, 0};
+        }
+    }
+    Generation generation(void) const override { return Generation::THREE; }
+    u16 maxCount(void) const override { return 0xFFFF; }
+    u16 id(void) const override { return ItemConverter::g3ToNational(Endian::convertTo<u16>(itemData.data())); }
+    void id(u16 v) override { Endian::convertFrom<u16>(itemData.data(), ItemConverter::nationalToG3(v)); }
```

Comment:
Do me a favor and mark these as necessary to review further (`TODO: Review and figure out what we want here` or something like that), that way we'll hopefully remember come back and figure out what we should do about the conversion problem